### PR TITLE
Change custom branding top logo configuration

### LINKED ISSUE #6774: Add custom top logo configuration documentation
# Description

Team, we need to add the custom top logo configuration documentation to explain the setting that allows to change it.

Related to https://github.com/wazuh/wazuh-packages/issues/2608

image

image

### PR DESCRIPTION
**Pull request by asteriscos**
## Description
Hi team, this pull request changes the custom branding configuration. It changes a few descriptions and adds a new way to change the top header logo in the `opensearch_dashboards.yml` configuration.

Closes #6774 

custom-branding-settings-health-check

### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/user-manual/wazuh-dashboard/custom-branding.rst [hunk 1]
```diff
@@ -16,13 +16,33 @@ Custom logos in the Wazuh dashboard
...
+-  ``customization.logo.healthcheck``. This property sets the `Health check logo` image. It has a size limit of 1 MB. It replaces the logo on top of the check list displayed during the health check routine. Recommended size: 300 pixels width, 70 pixels height.
 
-.. thumbnail:: /images/kibana-app/features/white-labeling/custom-branding-settings.jpg
+.. thumbnail:: /images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png
    :align: center
-   :width: 80%
+   :width: 100%
+
+
+To customize the `Wazuh dashboard home logo` in the top header, you have to edit the file ``/usr/share/wazuh-dashboard/config/opensearch_dashboards.yml`` and add the URL of the default and dark theme logos.
```
Comment (Desvelao) [inline source/user-manual/wazuh-dashboard/custom-branding.rst:32]: **nitpick**: The configuration file in the `.rpm`, and `.deb` is located at `/etc/wazuh-dashboard/opensearch_dashboards.yml` (see https://documentation.wazuh.com/current/installation-guide/wazuh-dashboard/step-by-step.html#configuring-the-wazuh-dashboard) while the Docker distribution could have using the `/usr/share/wazuh-dashboard/config/opensearch_dashboards.yml` (see https://github.com/wazuh/wazuh-docker/blob/4.8.0/build-docker-images/wazuh-dashboard/config/entrypoint.sh#L20). We should review it.

**Review by Desvelao — APPROVED** [review]
LGTM!

File: source/images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png
Comment (javimed) [inline source/images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png:1]: We could add a square surrounding the logo image we're customizing

File: source/images/kibana-app/features/white-labeling/custom-branding-settings-loading.png
Comment (javimed) [inline source/images/kibana-app/features/white-labeling/custom-branding-settings-loading.png:1]: We could add a square surrounding the logo image we're customizing

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/images/kibana-app/features/white-labeling/custom-branding-settings-loading.png
Comment (asteriscos) [reply source/images/kibana-app/features/white-labeling/custom-branding-settings-loading.png:1]: done!

File: source/images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png
Comment (asteriscos) [reply source/images/kibana-app/features/white-labeling/custom-branding-settings-health-check.png:1]: done!

**Review by javimed — APPROVED** [review]

**Pull request merged**